Makefile

```diff
@@ -51,3 +51,8 @@ build-ci-container:
 .PHONY: login-to-ecr upload-ci-container
 push-ci-container:
 	docker push 320464205386.dkr.ecr.us-west-2.amazonaws.com/custom-codebuild-cis-ci
+
+.PHONY: test
+test:
+	docker-compose build
+	docker-compose run tester bash -c '/root/utils/fake-creds.sh && source /root/.bashrc && make -j4 test-tox'
```

Comment:
because this can make a huge difference in this case - quick optimization:
```
make -j $(getconf _NPROCESSORS_ONLN)
```

or maybe nicer:
```
NPROC = $(shell getconf _NPROCESSORS_ONLN)

....
... make -j $(NPROC)
```